client/blocks/login/two-factor-authentication/security-key-form.jsx

```diff
@@ -51,9 +56,13 @@ class SecurityKeyForm extends Component {
 								} ) }
 							</p>
 							<p>
-								{ translate(
-									'Insert your security key into your USB port. Then tap the button or gold disc.'
-								) }
+								{ isWoo
+									? translate(
+											'Insert your security key into your USB port, then tap the button or gold disc.'
```

Comment:
@verofasulo It seems Woo and non-Woo text is slightly different:

`Insert your security key into your USB port, then tap the button or gold disc.` <-- Woo
`Insert your security key into your USB port. Then tap the button or gold disc.` <-- Non-woo

Would it be better if we standardise this?

Reply:
@ilyasfoo It's Larua's call. I asked a similar question here - OQjzEJlmdgfMzBZxwnCbUn-fi ?node-id=0%3A1

Reply:
Gotcha, @chihsuan! Thanks for clarifying. Closing this comment